### DISCUSSION TITLE
How to link the features shown in the current bounding box along with their properties?

### DISCUSSION THREAD

**Discussion by ThibautAr** [geopython/pygeoapi#850] [Q&A]
Hello,

I would like to know if there is a way to link the data shown on the bounding box (left) with their properties (right).
For instance, for the attached image, I would like to have only three rows (right) showing the properties of the three features (left).

![image](https://user-images.githubusercontent.com/98975549/152532355-b6d00359-9066-494e-9dd4-40e058a86103.png)

I'm guessing that it should be in [https://github.com/geopython/pygeoapi/blob/master/pygeoapi/templates/collections/items/index.html](https://github.com/geopython/pygeoapi/blob/master/pygeoapi/templates/collections/items/index.html) but I don't know what/where to modify.

Thanks

**Comment by tomkralidis** [comment]
In general, the JavaScript would need to listen to the map events and update the table based on the features visible in the map.
